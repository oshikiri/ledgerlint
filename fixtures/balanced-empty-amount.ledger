2020-03-26 * super market
  Expenses:Household essentials  200 JPY
  Expenses:Food  600 JPY
  Assets:Cash
