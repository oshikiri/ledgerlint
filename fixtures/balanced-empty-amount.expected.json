{"type":"decoration","source":"ledgerlint","file_path":"fixtures/balanced-empty-amount.ledger","range":{"start":{"line":3,"character":0},"end":{"line":3,"character":80}},"renderOptions":{"after":{"contentText":"- 800 JPY","color":"grey","margin":"2ch"}}}
